config/deals.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+type DealsConfig struct {
+	FailOnTransferFailure bool `json:",omitempty"`
```

Comment:
`QuitOnTransferFailure` ?

Reply:
I have no earthly idea what this actually does, so no opinion. Quit would imply to me that the whole system shuts down if a transfer fails... is that the case here?